WalletWasabi.Gui/Controls/WalletExplorer/TransactionBroadcasterViewModel.cs

```diff
@@ -165,20 +166,26 @@ private async Task OnDoTransactionBroadcastAsync()
 				ButtonText = "Broadcasting Transaction...";
 
 				SmartTransaction transaction;
-				try
-				{
-					var signedPsbt = PSBT.Parse(TransactionString, Global.Network ?? Network.Main);
 
+				if (PSBT.TryParse(TransactionString, Global.Network ?? Network.Main, out var signedPsbt))
+				{
 					if (!signedPsbt.IsAllFinalized())
 					{
 						signedPsbt.Finalize();
 					}
 
 					transaction = signedPsbt.ExtractSmartTransaction();
 				}
-				catch
+				else
 				{
-					transaction = new SmartTransaction(Transaction.Parse(TransactionString, Global.Network ?? Network.Main), WalletWasabi.Models.Height.Unknown);
+					if (Transaction.TryParse(TransactionString, Global.Network, out var txn))
```

Comment:
I think this is fine, but another alternative is to just use `Transaction.Parse` that will also have a `FormatException` anyway, that would save you the `else` block.